PYME/Acquire/SpoolController.py

```diff
@@ -552,7 +552,10 @@ def start_spooling(self, fn=None, settings={}, preflight_mode='interactive'):
             backend_kwargs['compression_settings'] = settings.get('pzf_compression_settings', self.pzf_compression_settings)
         elif self.spoolType == 'File':
             backend_kwargs['complevel'] = settings.get('hdf_compression_level', self.hdf_compression_level)
-        
+        # set dimensions and shape
```

Comment:
This will need to be revisited to make sure it plays well with `XYZTCAcquisition`and the like. `XYZTCAcquisition creates it's backend with `backend(..., shape=shape, dim_order=dim_order, **backend_kwargs)` meaning that you will get repeated `dim_order` and `shape`  keyword arguments to the backend and an error.